contrib/seeds/makeseeds.py

```diff
@@ -30,7 +30,7 @@
 PATTERN_IPV4 = re.compile(r"^((\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})):(\d+)$")
 PATTERN_IPV6 = re.compile(r"^\[([0-9a-z:]+)\]:(\d+)$")
 PATTERN_ONION = re.compile(r"^([abcdefghijklmnopqrstuvwxyz234567]{16}\.onion):(\d+)$")
-PATTERN_AGENT = re.compile(r"^(/Satoshi:0.14.(0|1|2|99)/|/Satoshi:0.15.(0|1|2|99)|/Satoshi:0.16.(0|1|2|99)/)$")
+PATTERN_AGENT = re.compile(r"^(/Satoshi:0.14.(0|1|2|3|99)/|/Satoshi:0.15.(0|1|2|99)|/Satoshi:0.16.(0|1|2|3|99)/|/Satoshi:0.17.(0|1|2|99)/|/Satoshi:0.18.99/)$")
```

Comment:
We should add an instruction to add new version here in /contrib/seeds/README.md.

Reply:
There is one:

> Be sure to update `PATTERN_AGENT` in `makeseeds.py` to include the current version,
and remove old versions as necessary (at a minimum when GetDesirableServiceFlags
changes its default return value, as those are the services which seeds are added
to addrman with).